src/main/java/graphql/execution/instrumentation/Instrumentation.java

```diff
@@ -2,10 +2,10 @@
 
 import graphql.ExecutionResult;
 import graphql.execution.instrumentation.parameters.DataFetchParameters;
-import graphql.execution.instrumentation.parameters.ExecutionParameters;
 import graphql.execution.instrumentation.parameters.FieldFetchParameters;
 import graphql.execution.instrumentation.parameters.FieldParameters;
```

Comment:
Do you want to rename `DataFetchParameters`, `FieldFetchParameters`, and `FieldParameters` while you're at it?